static/js/pages/independent-expenditures.js

```diff
@@ -0,0 +1,83 @@
+'use strict';
+
+var $ = require('jquery');
+var _ = require('underscore');
+
+var tables = require('../modules/tables');
+var helpers = require('../modules/helpers');
+var columns = require('../modules/columns');
+
+var FilterPanel = require('fec-style/js/filter-panel').FilterPanel;
+var filterTags = require('fec-style/js/filter-tags');
+
+var expenditureTemplate = require('../../templates/independent-expenditures.hbs');
+
+var columns = [
+  {
+    data: 'committee',
+    orderable: false,
+    className: 'all',
+    render: function(data, type, row, meta) {
+      if (data) {
+        return tables.buildEntityLink(
+          data.name,
+          helpers.buildAppUrl(['committee', data.committee_id]),
+          'committee'
+        );
+      } else {
+        return '';
+      }
+    }
+  },
+  tables.currencyColumn({data: 'expenditure_amount', className: 'min-tablet'}),
+  {
+    data: 'candidate_name',
+    orderable: false,
+    className: 'min-tablet hide-panel',
+    render: function(data, type, row, meta) {
+      if (row.candidate_id) {
+        return tables.buildEntityLink(
+          data,
+          helpers.buildAppUrl(['candidate', row.candidate_id], tables.getCycle(row, meta)),
+          'candidate'
+        );
+      } else {
+        return row.candidate_name;
+      }
+    }
+  },
+  _.extend({}, columns.supportOpposeColumn, {className: 'min-tablet'}),
+  tables.dateColumn({data: 'expenditure_date', className: 'min-desktop hide-panel-tablet'}),
+  tables.urlColumn('pdf_url', {data: 'expenditure_description', className: 'min-desktop hide-panel', orderable: false}),
+  {
+    className: 'all u-no-padding',
+    width: '20px',
+    orderable: false,
+    render: function(data, type, row, meta) {
```

Comment:
'meta' is defined but never used.<br>'row' is defined but never used.<br>'type' is defined but never used.<br>'data' is defined but never used.